workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/enums/Order.ts

```diff
@@ -15,12 +15,24 @@ export enum ORDER_CUSTOM_FIELDS {
 
 export enum ORDER_TYPES {
 	ADDONS = 'ADDONS',
-	DXPAPP = 'DXPAPP',
+	CLIENT_EXTENSION = 'CLIENT_EXTENSION',
 	CLOUDAPP = 'CLOUDAPP',
+	DXPAPP = 'DXPAPP',
+	FRAGMENT = 'FRAGMENT',
 	SOLUTIONS7 = 'SOLUTIONS7',
 	SOLUTIONS30 = 'SOLUTIONS30',
 }
 
+export const ORDER_TYPES_LABELS: {[key: string]: string} = {
```

Comment:
No need to add the type since is automatically inferred here, what could be done is to add the `as const` to add the real keys and values as type

```suggestion
export const ORDER_TYPES_LABELS = {
```